common/build/build-common/tsc-multi.cjs.json

```diff
@@ -1,4 +1,4 @@
 {
-	"targets": [{ "extname": ".cjs", "module": "node16" }],
+	"targets": [{ "extname": ".cjs", "dExtName": ".d.cts", "module": "node16", "outDir": "./dist" }],
```

Comment:
Nit: you may want to make this change separately since it will affect more than just tree.

Reply:
reverted - I no longer think we should use tsc-multi; so, I'll just drop it.